lib/index.js

```diff
@@ -66,6 +90,14 @@ exports.register = (server, options, next) => {
       }
 
       return reply.continue();
+    })
+    .finally(() => {
+      if (options.timer) {
+        const diff = process.hrtime(startTime);
+        const milliseconds = diff[0] * MS_PER_SECOND + diff[1] / NS_PER_MS;
```

Comment:
Reading the [node docs](https://nodejs.org/docs/latest-v10.x/api/all.html#process_process_hrtime_time) it looks like the result of `process.hrtime` is a tuple of `[seconds, nanoseconds]`. If that's the case, wouldn't you want to multiple by `NS_PER_SECOND` to get everything into nanoseconds before summing them up?

Reply:
we could do that, and then convert nanoseconds to milliseconds.

i just copied this logic from https://github.com/lob/metrics-node/blob/957f8d99a4875878d59b125744af674657037dad/lib/timer.js#L19 which turns both the seconds and nanoseconds into milliseconds before summing them.